browser/ui/webui/brave_wallet/wallet_panel_ui.cc

```diff
@@ -92,9 +92,6 @@ WalletPanelUI::WalletPanelUI(content::WebUI* web_ui)
   source->AddBoolean(brave_wallet::mojom::kP3ACountTestNetworksLoadTimeKey,
                      base::CommandLine::ForCurrentProcess()->HasSwitch(
                          brave_wallet::mojom::kP3ACountTestNetworksSwitch));
-  if (ShouldDisableCSPForTesting()) {
-    source->DisableContentSecurityPolicy();
-  }
```

Comment:
Why this is no longer needed?

Reply:
Upstream has deleted this function [1] as not needed anymore. I had tried to reinsert it through a clever use of overrides, however I wasn't sure this was at all required, as none of the tests involved with this call were broken. This change is still marked as WIP because I couldn't get to a final decision on this.

[1] https://crrev.com/c/5328987

Reply:
> DisableContentSecurityPolicy was unused outside of tests (where it wasn't needed)

Not sure how to track implementation of "where it wasn't needed" part, but that seems fine if tests are green